src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3552,6 +3552,8 @@ void MusicXMLParserDirection::pedal(const QString& type, const int /* number */,
             p->setBeginText(u"<sym>keyboardPedalPed</sym>");
             p->setContinueText(u"(<sym>keyboardPedalPed</sym>)");
         } else {
+            p->setBeginText(u"");
+            p->setContinueText(u"");
```

Comment:
This is different from #20219

Reply:
Sorry, just ascreenshot from my mobile

Reply:
Exactly, this is just the fixing part that should go into 4.2.